### PR TITLE
Revised many parts of the account console that were not correctly translated into Chinese.

### PR DESCRIPTION
**Pull request by potterhe**

### CODE REVIEW

**Comment by ahus1** [comment]
Thank you for this contribution. Please have a second Chinese native speaker review your changes, and approve them here with a comment. Once the review is in, this PR can be merged. Thanks!

**Comment by jonkoops** [comment]
@xianli123 would you be willing to review this one?

**Comment by xianli123** [comment]
Thanks, @potterhe @ahus1 @jonkoops Yes. I will take a look. Thanks for doing that.

File: themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_zh_CN.properties [hunk 1]
```diff
@@ -1 +1,59 @@
...
+password-display-name=密码
+password-help-text=通过输入密码的方式登陆。
+otp-display-name=验证器应用程序
+otp-help-text=输入验证器应用程序中的验证码。
+basic-authentication=基本身份验证
+invalidRequestMessage=无效的请求
+
+# Applications page
+applicationsPageTitle=应用
+applicationsPageSubTitle=管理您的应用程序权限
```
Comment (xianli123) [inline themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_zh_CN.properties:48]:
```suggestion
applicationsPageSubTitle=查看您帐户有权访问的应用程序
```

File: themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_zh_CN.properties [hunk 2]
```diff
@@ -1 +1,59 @@
...
+internalApp=内置
```
Comment (xianli123) [inline themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_zh_CN.properties:49]:
```suggestion
internalApp=内置应用
```

**Comment by xianli123** [comment]
Hi @potterhe Thanks for your great contribution. Overall, it looks great. I just made a few tweaks, please take a look at the suggestions in the original file. If you have any opinion on the fine-tuning, we can discuss it.

**Comment by ahus1** [comment]
@xianli123 - I see that there have been some new commits to the PR. Could you please re-review and write a short note if your previous comments are now resolved?

**Review by xianli123 — COMMENTED** [review]
@ahus1 Previous commits have been resolved. It looks good for now.

**Comment by ahus1** [comment]
@xianli123 - thank you for the review. I see you've also added new review comments, so I assume this PR is not ready to be merged until those have been addressed by the original author?

**Comment by xianli123** [comment]
Hi @ahus1 Sorry for misleading you. I have reviewed the changes and approved this PR. So we can merge it for now. Thanks!

**Review by ahus1 — APPROVED** [review]
Approving as of community review

**Pull request merged**